integrations/prodigy_openai/openai/templates/ner_prompt.jinja2

```diff
@@ -0,0 +1,19 @@
+From the text below, extract the following entities in the following format:
+{# whitespace #}
+Cell: <comma delimited list of strings>
```

Comment:
Mainly asking out of curiosity: is there a reason you decided to write a custom prompt?

Reply:
This one's more for convenience when I was testing different templates. I find it a bit of a hassle typing each label in the CLI. Although the current `project.yml` setup already does that (but for now it's not being rendered into the template because there's no reference to `labels`). I'll update this later